### PR TITLE
doc: small documentation edits pre-CS2

### PR DESCRIPTION
**Pull request by FrancescoSer**
Edited 54H logging doc.

### CODE REVIEW

**Comment by FrancescoSer** [comment]
@hubertmis Do you know which core outputs logs as ``mod``, 0x24?

**Review by annwoj — APPROVED** [review]

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst [hunk 1]
```diff
@@ -155,15 +155,16 @@ To read the dictionary-based STM log output, do the following:
    * ``<app_name>`` is the application name.
    * ``<port>`` is the serial port used for output.
      Use ``nrfutil device list`` to list which serial ports are exposed by the development kit.
+   * The output can be either the console (``--stdout ascii``) or a file (the :file:`out.txt` file if ``--output-ascii out.txt``).
```
Comment (hubertmis) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst:158]: This command is confusing. "ascii" is a format, not an output target. I would expect `--out stdout` or `--out path_to_file`. But I guess it is as it is.

**Review by hubertmis — COMMENTED** [review]
I guess `mod` is short of "modem", so cellular CPUs in nRF9x series

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst [hunk 2]
```diff
@@ -155,15 +155,16 @@ To read the dictionary-based STM log output, do the following:
...
 
    nrfutil will capture the log data from the specified UART port and use the provided dictionary databases to decode the logs into a human-readable format.
-   The decoded logs will be saved in the specified output file (the :file:`out.txt` file in the previous example).
+   The decoded logs are sent to the previously-defined output (either the console or the :file:`out.txt` file in the previous example).
 
-#. Open the output file to review the decoded log messages.
+#. Read the terminal or open the output file to review the decoded log messages.
 
-   The file will contain timestamps and the log messages in a format that is human-readable.
+   The output will contain timestamps and the log messages in a format that is human-readable.
```
Comment (hubertmis) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst:167]: How about the present tense?
```suggestion
   The output contains timestamps and the log messages in a human-readable format.
```

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst (hunk 1 above)
Comment (nordic-piks) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst:158]: The help states:
```OUTPUT ARGUMENTS:
      --output-raw <PATH>
          Output the trace data to raw (binary) file format

      --output-ascii <PATH>
          Output the trace data to ascii file format (plain text)

      --stdout <OUTPUT_TYPES>
          Output the trace data to standard out.

          Select between one or more output types, separating each value with comma.

          Possible values:

          - binary:  Display binary data

          - network: Display network data

          - tls:     Display TLS data

          - ascii:   Display ASCII data

          Possible values:
          - binary:  Display binary data
          - network: Display network data
          - ascii:   Display ASCII data
          - tls:     Display TLS data
          -
```

**Review by nordic-piks — APPROVED** [review]

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst (hunk 1 above)
Comment (hubertmis) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst:158]: Yeah, these commands are confusing. As a user, I would expect:

`--format ascii|binary|network|tls` printing data to stdout. Then I could simply do `> path_to_file` to redirect the output stream. If console redirection is not enough, a common approach is `--output stdout|path_to_file` that can be used to redirect the stream from within the program.

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst (hunk 2 above)
Comment (FrancescoSer) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst]: fair point, fixed

**Comment by FrancescoSer** [comment]
> I guess `mod` is short of "modem", so cellular CPUs in nRF9x series

@hubertmis So do you think we remove that line?

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst:158]: Reported in https://nordicsemi.atlassian.net/browse/NRFU-1251

**Review by nordic-krch — APPROVED** [review]

**Pull request merged**